.github/workflows/update.yml

```
@@ -0,0 +1,152 @@
+name: Update vuln-list
+on: workflow_dispatch
+
+jobs:
+  update:
+    name: Update repo vuln-list
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
```

Let's bump up Go to 1.14.
```suggestion
    - name: Set up Go 1.14
      uses: actions/setup-go@v1
      with:
        go-version: 1.14
```